lib/device.c

```diff
@@ -546,6 +574,11 @@ int sync_update(struct sync_device *d, int row, struct sync_cb *cb,
 	return -1;
 }
 
+int sync_set_val(const struct sync_device *d, const struct sync_track *t, int row, float val)
+{
+	return send_set_key_cmd(d, t, row, val, KEY_STEP);
```

Comment:
Do we always want to use `KEY_STEP` here? I mean, it seems like a reasonable value, just making sure...

Reply:
KEY_STEP is just what I was using so far, now that I see you're receptive to the idea of supporting this feature and this isn't abandonware I'll polish up this PR and add setting the key_type while at it.